scalyr_agent/tests/configuration_k8s_test.py

```diff
@@ -372,3 +374,16 @@ def _test_k8s_ignore_namespaces_supersedes(env_str, local_str, expected):
         _test_k8s_ignore_namespaces_supersedes(None, None, kube_system)
         _test_k8s_ignore_namespaces_supersedes(None, r'""', [])
         _test_k8s_ignore_namespaces_supersedes(r'', r'""', [])
+
+
+class TestK8SUtils(TestConfigurationBase):
+    """
+    Tests for monitor_utils/k8s.py
+    """
+    def test_k8s_utils_qualified_name(self):
```

Comment:
Can we also have tests for checking when is_valid() is false - testing all combinations, e.g.  `namespace` None but valid `podname`, and `podname` None but valid `namespace`, and both `None`.

Likewise tests for == that test a valid Qualified name against a non-valid Qualified name.